Meta/lint-prettier.sh

```diff
@@ -10,20 +10,20 @@ if [ "$#" -eq "0" ]; then
         git ls-files \
             --exclude-from .prettierignore \
             -- \
-            '*.js' '*.mjs'
+            '*.js' '*.mjs' '*.md'
     )
 else
     files=()
     for file in "$@"; do
-        if [[ "${file}" == *".js" ]] || [[ "${file}" == *".mjs" ]]; then
+        if [[ "${file}" == *".js" ]] || [[ "${file}" == *".mjs" ]] || [[ "${file}" == *".md" ]]; then
```

Comment:
nit: can we just `[[ "${file}" == *".js" || "${file}" == *".mjs" || "${file}" == *".md" ]]`? either one is equal but the current version hurts to look at more